tensorbay/cli/auth.py

```diff
@@ -132,3 +135,21 @@ def _check_args_and_options(arg1: str, arg2: str, get: bool, unset: bool, is_all
 def _echo_formatted_profile(name: str, value: str) -> None:
     formatted_value = indent(value, INDENT)
     click.echo(f"{name} = {formatted_value}\n")
+
+
+def _check_auth(arg1: str, arg2: str, profile_name: str) -> None:
```

Comment:
Is it necessary to pass the profile name?
Can we add a default value to the `profile_name` argument in `get_gas` method?